### PR TITLE
cryptol 2.7.0

### PR DESCRIPTION
**Pull request by chenrui333**

### CODE REVIEW

**Comment by chenrui333** [comment]
```
18:39:20 ==> brew audit cryptol --online
18:40:24 ==> FAILED
18:40:24 ==> Installing 'bundler' gem
18:40:24 Successfully installed bundler-2.0.1
18:40:24 1 gem installed
18:40:24 Error: 1 problem in 1 formula detected
18:40:24 cryptol:
18:40:24   * The URL https://www.cryptol.net/ is not reachable
18:40:24 
18:40:24 ==> brew style cryptol
1
```

File: Formula/cryptol.rb [hunk 1]
```diff
@@ -4,9 +4,9 @@ class Cryptol < Formula
   include Language::Haskell::Cabal
 
   desc "Domain-specific language for specifying cryptographic algorithms"
-  homepage "https://www.cryptol.net/"
-  url "https://hackage.haskell.org/package/cryptol-2.6.0/cryptol-2.6.0.tar.gz"
-  sha256 "5f8abbfa2765ac0f6bb887edbec7032677d107c39581a4c78614e97382738f42"
+  homepage "https://github.com/GaloisInc/cryptol"
```
Comment (chenrui333) [inline Formula/cryptol.rb]: `https://www.cryptol.net/` is not reachable.

Comment (Moisan) [reply Formula/cryptol.rb]: Maybe they had an issue recently but it's back up. I think we should keep it as homepage.

**Review by Moisan — CHANGES_REQUESTED** [review]

**Comment by chenrui333** [comment]
Oh, interesting 🤣 I will revert my change then.

File: Formula/cryptol.rb (hunk 1 above)
Comment (chenrui333) [reply Formula/cryptol.rb]: I will revert this line change.

**Comment by chenrui333** [comment]
@Moisan I have just reverted the homepage change.

**Comment by fxcoudert** [comment]
@BrewTestBot test this please

**Comment by zbeekman** [comment]
Thanks for your contribution to Homebrew! 🎉 🥇 

Without awesome contributors like you, it would be impossible to maintain Homebrew to the high level of quality users have come to expect. Thank you!!!!

**Pull request closed without merging**